profile/authCallback.php

```diff
@@ -25,15 +28,15 @@
     $status = $oidc->authenticate();
   }
   catch (Exception $ex){
-    $_SESSION['last_message'] = 'Caught exception: ' . $ex->getMessage() . ' <ERR/>';
-    header('Location:' . $CLIENT_ROOT . '/profile/index.php');
+    $_SESSION['last_message'] = $LANG['CAUGHT_EXCEPTION'] . $ex->getMessage() . ' <ERR/>';
```

Comment:
It's generally best to not include spaces in the lang tags, as they're liable to get lost. Can you take it out of the lang tag and include it in the concatenated statement here?

Reply:
@Atticus29, I think these don't need lang tags, is that correct?

Reply:
@themerekat , after thinking about what this is actually doing, I realized that these DO need to be translated, actually. The 'last_message' session gets tracked and output to the end user in profile/index.php.